app/uk/gov/hmrc/teamsandrepositories/TeamRepositoryWrapper.scala

```diff
@@ -104,26 +104,46 @@ object TeamRepositoryWrapper {
 
     private case class RepositoriesToTeam(repositories: Seq[Repository], teamName: String)
 
-//    private def asNameListOfGivenRepoType(repoType: RepoType.Value): Seq[String] = {
-//      val repoNames = for {
-//        d <- teamRepos
-//        r <- extractRepositoryGroupForType(repoType, d.repositories)
-//      } yield r.name
-//
-//      repoNames
-//        .distinct
-//        .sortBy(_.toUpperCase)
-//    }
+    //    private def asNameListOfGivenRepoType(repoType: RepoType.Value): Seq[String] = {
```

Comment:
Any reason for keeping this? If so it's cool, but usually commented code is dead code!